faq/platforms.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Platforms
+
+order: 10
+---
+
+In this page we answer some common questions about supported platforms.
+
+## Platforms
+
+**Q: Which platforms are supported?**
+
+**A:** Currently the following platforms are supported:
+
+* macOS
+* iOS
+* MS-Windows
+* Android
```

Comment:
I just want to say, in case someone stumbles onto this PR, that this is not entirely correct. We currently support the following operating systems:
- FreeBSD
- NetBSD
- OpenBSD
- Linux
- Android
- macOS
- iOS
- Windows